packages/cli/lib/lib/webpack/sw_debug.js

```diff
@@ -0,0 +1,16 @@
+self.addEventListener('fetch', function(event){
+    const isJsonRequest = event.request.headers.get('content-type') === 'application/json';
+    const isPostRequest = event.request.method === 'POST';
+    event.respondWith(fetch(event.request).then(function(response){
+        return response;
+    }).catch(function(err){
+        console.error(err);
+        if (!navigator.onLine && isJsonRequest) {
+            if (isPostRequest) {
+                console.log('May be you should look into backgroundSync');
```

Comment:
Lets improve both messages like "PREACT-CLI-FRIENDLY-MESSAGE(IMPROVEMENT): We saw that your GET call failed. If this is due to a network error, may be you should look into runtimeCaching`.